opta/opta_run_check.py

```diff
@@ -0,0 +1,41 @@
+"""
+Usage: This file is used to check if Opta is already running in the current directory.
+This creates a file in opta directory with the name: .opta_running_<underscored-cwd>_<pid>
+"""
+
+
+import glob
+import os
+
+from opta.constants import HOME
+from opta.exceptions import UserErrors
+
+OPTA_RUNNING_LOC_PREFIX = os.path.join(
+    HOME, ".opta", f".opta_running_{os.getcwd().replace(os.path.sep, '_')}_"
+)
+
+
+def check_opta_running_file_exists() -> bool:
+    return glob.glob(OPTA_RUNNING_LOC_PREFIX + "*").__len__() > 0
+
+
+def create_opta_running_file() -> None:
+    with open(f"{OPTA_RUNNING_LOC_PREFIX}_{os.getpid()}", "w") as f:
+        f.write("")
+
+
+def remove_opta_running_file() -> None:
```

Comment:
We have some nice "handling" when user presses Ctrl+C; can we check if this file can be deleted when user presses Ctrl+C before opta exits?

Reply:
Let me test the `Ctrl + C` case. But it should be caught under any circumstances, and since the `opta_run_end` is running in finally block, every graceful exit will run this LOC. 
Although the question is do we need to handle ungraceful exits, such as killing the Process directly or something...
Or have some sort of release mechanism?

Reply:
Verified it. Works.